README.md

```diff
@@ -285,6 +285,7 @@ Services to securely store your Docker images.
 
 
 ## Monitoring & Logging
+* [CA Technologies Docker Monitoring](https://www.ca.com/us/products/docker-monitoring.html) - Agile Operations solutions from CA deliver the modern Docker monitoring businesses need to accelerate and optimize the performance of microservices and the dynamic Docker environments running them. Monitor both the Docker environment and apps that run inside them.
```

Comment:
@klinebch I assume after the free trial it's going to be a paid service. If so, please also add $$$ label after the project name as follows:

`[CA Technologies Docker Monitoring - $$$](https://www.ca.com/us/products/docker-monitoring.html)`